1.9/administration/installing/update-a-node.md

```diff
@@ -0,0 +1,99 @@
+---
+post_title: Updating Nodes
+menu_order: 801
+---
+
+You can update agent nodes in an active DC/OS cluster by using maintenance windows or terminate signal. Maintenance windows are the preferred method since this is generally more stable and less error prone.
+
+These steps are useful if you are downsizing a cluster, reconfiguring agent nodes, or moving a node to a new IP. When you change Mesos attributes (`⁠⁠⁠⁠/var/lib/dcos/mesos-slave-common`⁠⁠⁠⁠) or resources (⁠⁠⁠⁠`/var/lib/dcos/mesos-resources`⁠⁠⁠⁠), you must remove the agent node and re-register it with the master node under a new UUID. The master will then recognize the new attributes and resources specification.
+
+### Prerequisites:
+
+*   SSH installed and configured. This is required when using the terminate signal method.
+*   Access to the [Admin Router permissions](/docs/1.9/overview/architecture/components/#admin-router).
+
+# Updating nodes by using maintenance windows
+With maintenance windows you can drain multiple nodes at the same time from outside the cluster. SSH access is not required.
+
+You can define a maintenance schedule to evacuate your tasks prior to changing agent attributes or resources. ⁠⁠⁠All tasks that are running on the agent at that time are killed when you change agent attributes or resources. Mesos treats re-registered agents as new agents.
```

Comment:
The second and third sentences here were copied above, but left in place here... is that intentional?  (Doesn't seem necessary)